bin/deps.ml

```diff
@@ -0,0 +1,40 @@
+(*
+ * Copyright (c) 2018 Ulysse Gérard <ulysse@gtarides.com>
+ *
+ * Permission to use, copy, modify, and distribute this software for any
+ * purpose with or without fee is hereby granted, provided that the above
+ * copyright notice and this permission notice appear in all copies.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+ * WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+ * MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+ * ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+ * WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+ * ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+ * OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+ *)
+
+let run files =
+  List.iter (fun f ->
+    let doc = Mdx.parse_file
+      Mdx.Normal (* todo infer syntax, works only with md for now *)
```

Comment:
This should be now possible with #222

Reply:
I reverted the changes in #222 so you can modify `Mdx.parse_file` to make it call `Syntax.infer` if no syntax is provided.